docs/content/developers/release-management.md

```diff
@@ -57,8 +57,7 @@ The following “Repository secret” environment variables must be added to <ht
 * Update `ddev/ddev-webserver` to use the new version of `ddev/ddev-php-base` and push it with the proper tag.
 * Make sure the Docker images are all tagged and pushed.
 * Make sure [`pkg/versionconstants/versionconstants.go`](https://github.com/ddev/ddev/blob/master/pkg/versionconstants/versionconstants.go) is all set to point to the new images and tests have been run.
-* If the [`devcontainer-feature.json`](https://github.com/ddev/ddev/blob/master/.github/devcontainers/src/install-ddev/devcontainer-feature.json) (for GitHub Codespaces) needs to be updated, use the [`devcontainer` CLI](https://github.com/devcontainers/cli) and a GITHUB_TOKEN that has power to manage packages, like `https://github.com/settings/tokens/1121534855` (`Package management token - see https://ddev.readthedocs.io/en/latest/developers/release-management/#prerelease-tasks`)
-    * Change the version in `.github/devcontainers/src/install-ddev/devcontainer-feature.json`.
+* If the [`devcontainer-feature.json`](https://github.com/ddev/ddev/blob/master/.github/devcontainers/src/install-ddev/devcontainer-feature.json) (for GitHub Codespaces) needs to be updated, use the [`devcontainer` CLI](https://github.com/devcontainers/cli) and a `GITHUB_TOKEN` that has power to manage packages (`write:packages` scope, classic PAT), change the version in the [`devcontainer-feature.json`](https://github.com/ddev/ddev/blob/master/.github/devcontainers/src/install-ddev/devcontainer-feature.json) and run:
```

Comment:
This one is for developers so latest might be better, but don't bother worrying about it.

Reply:
I removed this part here:
```
like `https://github.com/settings/tokens/1121534855` (`Package management token - see https://ddev.readthedocs.io/en/latest/developers/release-management/#prerelease-tasks`)
```
because it didn't make sense, the token link is personal (404 for me) and the link to the latest docs points to the same paragraph where it is mentioned.